src/components/data-processing/DataIntegration/DataIntegration.jsx

```diff
@@ -77,7 +76,7 @@ const DataIntegration = (props) => {
     },
     elbow: {
       title: 'Elbow plot',
-      plotUuid: generateDataProcessingPlotUuid(null, filterName, 2),
+      plotUuid: generateDataProcessingPlotUuid(null, filterName, 1),
```

Comment:
`experimentId` has been removed form the properties of `ElbowPlot`, but we are still passing it.